examples/with-mongodb/util/mongodb.js

```diff
@@ -16,30 +16,42 @@ if (!MONGODB_DB) {
 
 /**
  * Global is used here to maintain a cached connection across hot reloads
- * in development. This prevents connections growing exponentiatlly
+ * in development. This prevents connections growing exponentially
  * during API Route usage.
  */
 let cached = global.mongo
-if (!cached) cached = global.mongo = {}
+
+if (!cached) {
+  cached = global.mongo = {}
+}
 
 export async function connectToDatabase() {
-  if (cached.conn) return cached.conn
+  if (cached.conn) {
+    return cached.conn
+  }
+
   if (!cached.promise) {
     const conn = {}
+
     const opts = {
       useNewUrlParser: true,
       useUnifiedTopology: true,
     }
+
     cached.promise = MongoClient.connect(MONGODB_URI, opts)
       .then((client) => {
         conn.client = client
+
         return client.db(MONGODB_DB)
       })
       .then((db) => {
         conn.db = db
+
         cached.conn = conn
       })
   }
+
   await cached.promise
+
```

Comment:
@LauraBeatris We only worry about changes requested by eslint and prettier. Code style changes like this aren't required and it's never going to be perfect for everybody (I for example don't like that much spacing), so don't worry too much about this in the future.

Reply:
Got it!